### PR TITLE
🌐 [i18n-KO] Translated `contributing.md` to Korean

### PR DESCRIPTION
**Pull request by mjk0618**
# What does this PR do?

Translated the `contributing.md` file of the documentation to Korean.
Thank you in advance for your review. 

Part of https://github.com/huggingface/transformers/issues/20179

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [ ] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)
@bolizabeth, @nuatmochoi, @heuristicwave, @mjk0618, @keonju2, @harheem, @HongB1, @junejae, @54data, @seank021, @augustinLib, @sronger, @TaeYupNoh, @kj021, @eenzeenee 

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review? (Final)

May you please review this PR? @sgugger, @ArthurZucker, @eunseojo

### CODE REVIEW

File: CONTRIBUTING_ko.md [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# 🤗 Transformers에 기여하기 [[contribute-to-transformers]]
+
+누구나 🤗 Transformers에 기여할 수 있으며, 우리는 모든 사람의 기여를 소중히 생각합니다. 코드 기여는 커뮤니티를 돕는 유일한 방법이 아닙니다. 질문에 답하거나 다른 사람을 도와 문서를 개선하는 것도 매우 가치가 있습니다.
+
+🤗 Transformers를 널리 알리는 것도 큰 도움이 됩니다! 블로그 게시글에 멋진 프로젝트들이 가능하게 한 🤗 Transformers 라이브러리에 대해 언급하거나, 도움이 되었을 때마다 Twitter에 알리거나, 저장소에 ⭐️ 를 표시하여 감사 인사를 전해주세요.
```
Comment (seank021) [inline CONTRIBUTING_ko.md:21]: 멋진 프로젝트들'이' -> 멋진 프로젝트들'을' 이 조금 더 자연스러울 것 같아요!

또, 어순도 '멋진 프로젝트들을 가능하게 한 🤗 Transformers 라이브러리에 대해 블로그 게시글에 언급하거나'가 조금 더 자연스럽게 읽히는 것 같아 제안드려 봅니다!

File: CONTRIBUTING_ko.md [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+```bash
+python src/transformers/commands/transformers_cli.py env
+```
+
+
+### 새로운 기능을 원하시나요? [[do-you-want-a-new-feature]]
+
+🤗 Transformers에서 사용하고 싶은 새로운 기능이 있다면, 다음 내용을 포함하여 이슈를 제출해 주세요:
+
+1. 이 기능이 필요한 *이유*는 무엇인가요? 라이브러리에 대한 문제나 불만과 관련이 있나요? 프로젝트에 필요한 기능인가요? 커뉴미니테 도움이 될 만한 기능인가요?
```
Comment (seank021) [inline CONTRIBUTING_ko.md:80]: '커뮤니티에' 오타 있습니다!

File: CONTRIBUTING_ko.md [hunk 3]
```diff
@@ -0,0 +1,331 @@
...
+
+4. 가상 환경에서 다음 명령을 실행하여 개발 환경을 설정하세요:
+
+   ```bash
+   pip install -e ".[dev]"
+   ```
+
+   만약 이미 가상 환경에 🤗 Transformers가 설치되어 있다면, `-e` 플래그를 사용하여 설치하기 전에 `pip uninstall transformers`로 제거해주세요.
+   
+   여러분의 운영체제에 따라서, 그리고 🤗 Transformers의 선택적 의존성의 수가 증가하면서, 이 명령이 실패할 수도 있습니다. 그럴 경우 사용하려는 딥 러닝 프레임워크(PyTorch, TensorFlow, 그리고/또는 Flax)를 설치한 다음 다음 명령을 실행해주세요:
```
Comment (seank021) [inline CONTRIBUTING_ko.md:144]: '설치한 다음 다음 명령을' 부분에서, 다음 다음이 연속되는 것이 자연스럽지 않은 것 같아서
첫 번째 '다음' 뒤에 쉼표를 두거나, 첫번째 '다음'을 '이후' 등의 어휘로 바꾸는 게 좋을 것 같습니다!

File: CONTRIBUTING_ko.md [hunk 4]
```diff
@@ -0,0 +1,331 @@
...
+* 예외가 발생한 경우 *전체* 트레이스백.
+* 스크린샷과 같이 도움이 될 것으로 생각되는 추가 정보를 첨부해 주세요.
+
+운영체제와 소프트웨어 버전을 자동으로 가져오려면 다음 명령을 실행하세요:
+
+```bash
+transformers-cli env
+```
+
+저장소의 루트 디렉토리에서도 같은 명령을 실행할 수 있습니다:
```
Comment (54data) [inline CONTRIBUTING_ko.md:69]: glossary에 따라 '디렉토리' -> '디렉터리'로 제안드립니다!
```suggestion
저장소의 루트 디렉터리에서도 같은 명령을 실행할 수 있습니다:
```

File: CONTRIBUTING_ko.md [hunk 5]
```diff
@@ -0,0 +1,331 @@
...
+
+   이러한 검사에 대해 자세히 알아보고 관련 문제를 해결하는 방법은 [Pull Request에 대한 검사](https://huggingface.co/docs/transformers/pr_checks) 가이드를 확인하세요.
+
+   만약 `docs/source` 디렉터리 아래의 문서를 수정하는 경우, 문서가 빌드될 수 있는지 확인하세요. 이 검사는 Pull Request를 열 때도 CI에서 실행됩니다. 로컬 검사를 실행하려면 문서 빌더를 설치해야 합니다:
+   
+   ```bash
+   pip install ".[docs]"
+   ```
+
+   저장소의 루트 디렉토리에서 다음 명령을 실행하세요:
```
Comment (54data) [inline CONTRIBUTING_ko.md:196]:
```suggestion
   저장소의 루트 디렉터리에서 다음 명령을 실행하세요:
```

File: CONTRIBUTING_ko.md [hunk 6]
```diff
@@ -0,0 +1,331 @@
...
+☐ 모든 공개 메서드는 유용한 기술문서를 가져야 합니다 (예를 들어 [`modeling_bert.py`](https://github.com/huggingface/transformers/blob/main/src/transformers/models/bert/modeling_bert.py) 참조).<br>
+☐ 저장소가 빠르게 성장하고 있으므로 저장소에 상당한 부담을 주는 이미지, 동영상 및 기타 텍스트가 아닌 파일은 추가하지 마세요. 대신 [`hf-internal-testing`](https://huggingface.co/hf-internal-testing)과 같은 Hub 저장소를 사용하여 이러한 파일을 호스팅하고 URL로 참조하세요. 문서와 관련된 이미지는 다음 저장소에 배치하는 것을 권장합니다: [huggingface/documentation-images](https://huggingface.co/datasets/huggingface/documentation-images). 이 데이터셋 저장소에서 PR을 열어서 Hugging Face 멤버에게 병합을 요청할 수 있습니다.
+
+Pull Request에서 실행되는 검사에 대한 자세한 정보는 [Pull Request에 대한 검사](https://huggingface.co/docs/transformers/pr_checks) 가이드를 확인하세요.
+
+### 테스트 [[tests]]
+
+라이브러리 동작과 여러 예제를 테스트할 수 있는 광범위한 테스트 스위트가 포함되어 있습니다. 라이브러리 테스트는 [tests](https://github.com/huggingface/transformers/tree/main/tests) 폴더에, 예제 테스트는 [examples](https://github.com/huggingface/transformers/tree/main/examples) 폴더에 있습니다.
+
+속도가 빠른 `pytest`와 `pytest-xdist`를 선호합니다. 저장소의 루트 디렉토리에서 테스트를 실행할 *하위 폴더 경로 또는 테스트 파일 경로*를 지정하세요.
```
Comment (54data) [inline CONTRIBUTING_ko.md:252]:
```suggestion
속도가 빠른 `pytest`와 `pytest-xdist`를 선호합니다. 저장소의 루트 디렉터리에서 테스트를 실행할 *하위 폴더 경로 또는 테스트 파일 경로*를 지정하세요.
```

File: CONTRIBUTING_ko.md [hunk 7]
```diff
@@ -0,0 +1,331 @@
...
+
+```bash
+python -m pytest -n auto --dist=loadfile -s -v ./tests/models/my_new_model
+```
+
+마찬가지로 `examples` 디렉토리에서도 *하위 폴더 경로 또는 테스트 파일 경로*를 지정하세요. 예를 들어, 다음 명령은 PyTorch `examples` 디렉토리의 텍스트 분류 하위 폴더를 테스트합니다:
```
Comment (54data) [inline CONTRIBUTING_ko.md:258]:
```suggestion
마찬가지로 `examples` 디렉터리에서도 *하위 폴더 경로 또는 테스트 파일 경로*를 지정하세요. 예를 들어, 다음 명령은 PyTorch `examples` 디렉터리의 텍스트 분류 하위 폴더를 테스트합니다:
```

File: CONTRIBUTING_ko.md (hunk 3 above)
Comment (54data) [inline CONTRIBUTING_ko.md:144]: glossary에 따라 '딥 러닝' -> '딥러닝'으로 제안드립니다!
```suggestion
   여러분의 운영체제에 따라서, 그리고 🤗 Transformers의 선택적 의존성의 수가 증가하면서, 이 명령이 실패할 수도 있습니다. 그럴 경우 사용하려는 딥러닝 프레임워크(PyTorch, TensorFlow, 그리고/또는 Flax)를 설치한 다음 다음 명령을 실행해주세요:
```

File: CONTRIBUTING_ko.md [hunk 8]
```diff
@@ -0,0 +1,331 @@
...
+☐ Pull Request 제목은 기여 내용을 요약해야 합니다.<br>
+☐ Pull Request가 이슈를 해결하는 경우, Pull Request 설명에 이슈 번호를 언급하여 연관되어 있음을 알려주세요. (이슈를 확인하는 사람들이 해당 이슈에 대한 작업이 진행 중임을 알 수 있게 합니다).<br>
+☐ 작업이 진행중이라면 제목 앞에 `[WIP]`를 붙여주세요. 중복 작업을 피하고 병합할 준비가 된 PR과 구분하기에 유용합니다.<br>
+☐ 기존 테스트를 통과하는지 확인하세요.<br>
+☐ 새로운 기능을 추가하는 경우, 해당 기능에 대한 테스트도 추가하세요.<br>
+   - 새 모델을 추가하는 경우, `ModelTester.all_model_classes = (MyModel, MyModelWithLMHead,...)`을 사용하여 일반적인 테스트를 활성화하세요.
+   - 새 `@slow` 테스트를 추가하는 경우, 다음 명령으로 테스트를 통과하는지 확인하세요: `RUN_SLOW=1 python -m pytest tests/models/my_new_model/test_my_new_model.py`.
+   - 새 토크나이저를 추가하는 경우, 테스트를 작성하고 다음 명령으로 테스트를 통과하는지 확인하세요: `RUN_SLOW=1 python -m pytest tests/models/{your_model_name}/test_tokenization_{your_model_name}.py`. CircleCI에서는 느린 테스트를 실행하지 않지만, GitHub Actions에서는 매일 밤 실행됩니다!<br>
+
+☐ 모든 공개 메서드는 유용한 기술문서를 가져야 합니다 (예를 들어 [`modeling_bert.py`](https://github.com/huggingface/transformers/blob/main/src/transformers/models/bert/modeling_bert.py) 참조).<br>
```
Comment (54data) [inline CONTRIBUTING_ko.md:243]: glossary에 따라 '메서드' -> '메소드'로 제안드립니다!
```suggestion
☐ 모든 공개 메소드는 유용한 기술문서를 가져야 합니다 (예를 들어 [`modeling_bert.py`](https://github.com/huggingface/transformers/blob/main/src/transformers/models/bert/modeling_bert.py) 참조).<br>
```

File: CONTRIBUTING_ko.md [hunk 9]
```diff
@@ -0,0 +1,331 @@
...
+
+   ```bash
+   git push -u origin a-descriptive-name-for-my-changes
+   ```
+
+   이미 PR을 열었다면, `--force` 플래그와 함께 강제 푸시해야 합니다. 아직 PR이 열리지 않았다면 정상적으로 변경 사항을 푸시하면 됩니다.
+
+6. 이제 GitHub에서 포크한 저장소로 이동하고 **Pull request(풀 리퀘스트)**를 클릭하여 Pull Request를 열 수 있습니다. 아래의 [체크리스트](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md/#pull-request-checklist)에서 모든 항목에 체크 표시를 하세요. 준비가 완료되면 프로젝트 관리자에게 변경 사항을 보내 검토를 요청할 수 있습니다.
+
+7. 관리자가 변경 사항을 요청해도 괜찮습니다. 숙련된 기여자들도 동일한 상황을 겪습니다! 모두가 변경 사항을 Pull Request에서 볼 수 있도록, 로컬 브랜치에서 작업하고 변경 사항을 포크한 저장소로 푸시하세요. 그러면 변경 사항이 자동으로 Pull Request에 나타납니다.
```
Comment (nuatmochoi) [inline CONTRIBUTING_ko.md:230]:
```suggestion
7. 관리자가 변경 사항을 요청해도 괜찮습니다. 핵심 기여자들도 동일한 상황을 겪습니다! 모두가 변경 사항을 Pull Request에서 볼 수 있도록, 로컬 브랜치에서 작업하고 변경 사항을 포크한 저장소로 푸시하세요. 그러면 변경 사항이 자동으로 Pull Request에 나타납니다.
```
원문(core)을 살려 `핵심`으로 수정하였습니다.

File: CONTRIBUTING_ko.md [hunk 10]
```diff
@@ -0,0 +1,331 @@
...
+### Pull Request 체크리스트 [[pull-request-checklist]]
+
+☐ Pull Request 제목은 기여 내용을 요약해야 합니다.<br>
+☐ Pull Request가 이슈를 해결하는 경우, Pull Request 설명에 이슈 번호를 언급하여 연관되어 있음을 알려주세요. (이슈를 확인하는 사람들이 해당 이슈에 대한 작업이 진행 중임을 알 수 있게 합니다).<br>
+☐ 작업이 진행중이라면 제목 앞에 `[WIP]`를 붙여주세요. 중복 작업을 피하고 병합할 준비가 된 PR과 구분하기에 유용합니다.<br>
+☐ 기존 테스트를 통과하는지 확인하세요.<br>
+☐ 새로운 기능을 추가하는 경우, 해당 기능에 대한 테스트도 추가하세요.<br>
+   - 새 모델을 추가하는 경우, `ModelTester.all_model_classes = (MyModel, MyModelWithLMHead,...)`을 사용하여 일반적인 테스트를 활성화하세요.
+   - 새 `@slow` 테스트를 추가하는 경우, 다음 명령으로 테스트를 통과하는지 확인하세요: `RUN_SLOW=1 python -m pytest tests/models/my_new_model/test_my_new_model.py`.
+   - 새 토크나이저를 추가하는 경우, 테스트를 작성하고 다음 명령으로 테스트를 통과하는지 확인하세요: `RUN_SLOW=1 python -m pytest tests/models/{your_model_name}/test_tokenization_{your_model_name}.py`. CircleCI에서는 느린 테스트를 실행하지 않지만, GitHub Actions에서는 매일 밤 실행됩니다!<br>
```
Comment (nuatmochoi) [inline CONTRIBUTING_ko.md:241]:
```suggestion
   - 새 토크나이저를 추가하는 경우, 테스트를 작성하고 다음 명령으로 테스트를 통과하는지 확인하세요: `RUN_SLOW=1 python -m pytest tests/models/{your_model_name}/test_tokenization_{your_model_name}.py`. 
   - CircleCI에서는 느린 테스트를 실행하지 않지만, GitHub Actions에서는 매일 밤 실행됩니다!<br>
```
줄바뀜 누락된 부분 수정합니다.

**Review by nuatmochoi — CHANGES_REQUESTED** [review]

File: CONTRIBUTING_ko.md (hunk 3 above)
Comment (mjk0618) [reply CONTRIBUTING_ko.md:144]: 리뷰해주셔서 감사합니다! 말씀해주신 내용 반영해서 commit하였습니다!

File: CONTRIBUTING_ko.md (hunk 4 above)
Comment (mjk0618) [reply CONTRIBUTING_ko.md]: 리뷰해주셔서 감사합니다. 말씀하신 내용 반영하여 커밋하였습니다!

File: CONTRIBUTING_ko.md (hunk 9 above)
Comment (mjk0618) [reply CONTRIBUTING_ko.md]: 리뷰해주셔서 감사합니다! 말씀하신 내용 모두 반영해서 커밋하였습니다.

처음에 NMT Draft가 생성될 때 핵심 기여자로 되어 있었는데, 조금 더 이해가 쉽도록 숙련된으로 표현을 수정했었습니다. 다시 검토해보니 원문을 살리는 게 나을 거 같아서 반영해서 수정하였습니다!

**Review by stevhliu — COMMENTED** [review]
Thanks for the translation! 🤗 

Can you copy the contents of `CONTRIBUTING_ko.md` into `contributing.md`? We should just have one file for this.

**Comment by stevhliu** [comment]
Oh nevermind, I see what you mean! You can feel free to mark the PR as ready for review and we can merge!

**Comment by mjk0618** [comment]
> Oh nevermind, I see what you mean! You can feel free to mark the PR as ready for review and we can merge!

Thank you for your help! I marked the PR as ready for review. Could you please review and merge it?

**Comment by mjk0618** [comment]
May you please review this PR? @sgugger, @ArthurZucker, @eunseojo

**Comment by stevhliu** [comment]
@ArthurZucker, I think the `build_pr_documentation` test is failing because `docs/source/ko/contributing.md` is just a path to the actual docs and the [`generate_frontmatter_in_text`](https://github.com/huggingface/doc-builder/blob/c8152d4d1cb455e4b614dd34806f9c1345ad92b1/src/doc_builder/build_doc.py#L273) function is expecting a level 1 header. Would it still be ok to merge this PR?

**Comment by ArthurZucker** [comment]
@stevhliu would be better if we find a fix! It's my first time seeing this so not really sure

**Comment by mjk0618** [comment]
@stevhliu Thank you for reviewing my PR! Then, should I try to fix something so that the error does not occur? I think I have to  try to find out the reason for the error, if I can help fixing the problem. If it's something I can't do, would it be better to wait while translating other documents?

**Comment by stevhliu** [comment]
I think it's an issue with the test not recognizing the path to `contributing_ko.md` in the docs. We'll look into this issue, but in the meantime, feel free to keep up the good work translating other docs. This issue is probably only affecting this specific doc. 🙂

File: docs/source/ko/contributing.md [hunk 11]
```diff
@@ -0,0 +1 @@
+../../../CONTRIBUTING_ko.md
```
[the remark this answers is not in the document]
Comment (mjk0618) [reply docs/source/ko/contributing.md:1]: Before I make a new commit, I wanted to confirm if I unerstood correctly. Then, should I delete the `CONTRIBUTING_ko.md` file I added to the root directory, and copy all the contents to `docs/source/ko/contributing.md` file?

[the remark this answers is not in the document]
Comment (mjk0618) [reply docs/source/ko/contributing.md]: Thank you for your help! I've just finished what I should do.

**Review by stevhliu — APPROVED** [review]
Great work! Thanks for being patient and working with us on this 😄

File: docs/source/ko/contributing.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (stevhliu) [reply docs/source/ko/contributing.md]: Nothing left to do except merge! 🚀

**Pull request merged**